<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>671</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>120</y>
           <width>101</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>弹窗</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>101</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>进度条Bar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>111</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>进度条Dilog</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>118</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>200</y>
           <width>111</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>200</y>
           <width>121</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>280</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>事件过滤测试</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>120</y>
           <width>89</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string> 不规则窗体</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <widget class="QCustomPlot" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>641</width>
           <height>341</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Page</string>
        </attribute>
        <widget class="QwtPlot" name="qwtPlot">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>641</width>
           <height>341</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
